File: app/src/main/​src/main/java/com/trikego/app/MainActivity.java
package com.trikego.app;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;
import android.view.Gravity;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        TextView text = new TextView(this);
        text.setText("TrikeGo Sariaya\n\nApp started successfully!");
        text.setTextSize(24);
        text.setGravity(Gravity.CENTER);

        setContentView(text);
    }
}

File: app/src/main/​src/main/java/com/trikego/app/MainActivity.kt

package com.trikego.app

import android.app.Activity
import android.os.Bundle
import android.graphics.Color
import android.view.Gravity
import android.widget.TextView

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        try {
            super.onCreate(savedInstanceState)

            val tv = TextView(this)
            tv.text = "TrikeGo Sariaya\n\nApp started successfully!"
            tv.textSize = 24f
            tv.setTextColor(Color.BLACK)
            tv.gravity = Gravity.CENTER

            setContentView(tv)

        } catch (e: Exception) {
            val error = TextView(this)
            error.text = "TRIKEGO ERROR\n\n${e.javaClass.name}\n\n${e.message}"
            error.textSize = 18f
            error.setTextColor(Color.RED)
            error.gravity = Gravity.CENTER
            error.setPadding(30, 30, 30, 30)

            setContentView(error)
        }
    }
}
